scripts/india_edm/IndiaEnergy_Gas/IndiaEnergy_Gas.mcf

```diff
@@ -0,0 +1,167 @@
+Node: dcid:Annual_Consumption_Fuel_ElectricityGeneration_NaturalGas
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Fuel
+measuredProperty: dcs:consumption
+measurementQualifier: dcs:Annual
+statType: dcs:measuredValue
+energySource: dcs:NaturalGas
+consumingSector: dcs:ElectricityGeneration
+
+Node: dcid:Annual_Consumption_Fuel_Industry_NaturalGas
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Fuel
+measuredProperty: dcs:consumption
+measurementQualifier: dcs:Annual
+statType: dcs:measuredValue
+energySource: dcs:NaturalGas
```

Comment:
Can you pls use the property `fuelType: dcs:NaturalGas` since populationType is Fuel.
This will also match existing statvars like:
[Annual_Consumption_Fuel_Households_NaturalGas`](https://autopush.datacommons.org/browser/Annual_Consumption_Fuel_Households_NaturalGas)

Reply:
Added the `fuelType` property for Gas [mcf](https://github.com/datacommonsorg/data/blob/5210c3d67ac59d7ed771fff3330e8068f3e6f74e/scripts/india_edm/IndiaEnergy_Gas/IndiaEnergy_Gas.mcf#L7).

Also, resolved your other comments - removed `dc_generated/` folders and zipped all data into one file. Also added a functionality in the code that will automatically unzip the data files and use it in the preprocessing.